transactions/dkg/send_empty_final_submission.cdc

```diff
@@ -0,0 +1,20 @@
+import FlowDKG from "FlowDKG"
+
+// Used by DKG participants to submit their final submission for the current DKG instance.
+// This transaction is used when the participant locally failed the DKG process, and are
+// recording this by submitting an empty result. For non-empty submissions, use "send_final_submission".
+transaction() {
+
+    let dkgParticipant: &FlowDKG.Participant
+    let submission: FlowDKG.ResultSubmission
+
+    prepare(signer: auth(BorrowValue) &Account) {
+        self.dkgParticipant = signer.storage.borrow<&FlowDKG.Participant>(from: FlowDKG.ParticipantStoragePath)
+            ?? panic("Cannot borrow dkg participant reference")
```

Comment:
```suggestion
            ?? panic("Cannot borrow DKG Participant Reference from path "
                    .concat(FlowDKG.ParticipantStoragePath)
                    .concat(". The signer needs to make sure their account is initialized with the DKG Participant resource."))
```

It would be good to start updating the error messages here too. This would apply to most of the DKG transactions here too